src/Chart.php

```diff
@@ -230,15 +230,17 @@ public function setCurrencyLabel(string $char = '€', string $axis = 'y', int $
                     'mode' => 'point',
                     'callbacks' => [
                         'label' => new JsFunction(['context'], [
-                            new JsExpression(<<<'EOF'
-                                let label = context.dataset.label || "";
-                                // let value = context.parsed.y; // or x (horizontal) or r (radar) etc
-                                let value = context.formattedValue.replace(/,/, "");
-                                if (label) {
-                                    label += ": ";
-                                }
-                                return label + (value ? "' . $char . ' " +  Number(value).toLocaleString(undefined, {minimumFractionDigits: ' . $digits . ', maximumFractionDigits: ' . $digits . '}) : "No Data");
-                                EOF),
+                            new JsExpression(
```

Comment:
`new JsExpression(<<<'EOF' ... [] ... EOF, [php params])`

Reply:
Thanks for your quick feedback - fixed in the next commit